<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FGESelectionModelsForm</class>
 <widget class="QWidget" name="FGESelectionModelsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>177</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid ;
	border-color: rgb(156, 156, 156);
    margin-top: 10px;
    font-size: 14px;
    border-radius: 3px;
}

QGroupBox::title {
	top:0px;
	left : 15px;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px 2px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>161</height>
         </size>
        </property>
        <property name="title">
         <string>Type Selection :</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton_pick">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Tweak</string>
           </property>
           <property name="icon">
            <iconset resource="../../icons.qrc">
             <normaloff>:/Icons/svg/tweak1_selection.svg</normaloff>:/Icons/svg/tweak1_selection.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButton_circle">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Circle</string>
           </property>
           <property name="icon">
            <iconset resource="../../icons.qrc">
             <normaloff>:/Icons/svg/circle1_selection.svg</normaloff>:/Icons/svg/circle1_selection.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButton_lasso">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Lasso</string>
           </property>
           <property name="icon">
            <iconset resource="../../icons.qrc">
             <normaloff>:/Icons/svg/lasso1_selection.svg</normaloff>:/Icons/svg/lasso1_selection.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButton_rect">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Rectangle</string>
           </property>
           <property name="icon">
            <iconset resource="../../icons.qrc">
             <normaloff>:/Icons/svg/rect1_selection.svg</normaloff>:/Icons/svg/rect1_selection.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_close">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
